containerize/Containerfile

```diff
@@ -57,6 +58,23 @@ RUN mkdir -p /tmp/redocly/node_modules && if [ "$PREFETCH" == "true" ]; then \
     npm install --prefix /tmp/redocly; \
   fi
 
+### Validate installed dependency versions. This is crucial when dependencies are prefetched using Cachi2
```

Comment:
I don't love this approach of adding validation commands to a Containerfile, as it introduces extra image layers during a build. That said, the only alternative I can think of right now is another e2e test and that feels even more expensive.